### PR TITLE
Add Skinning Contest #3 Results news post

### PR DESCRIPTION
**Pull request by RockRoller01**
**Please do not spoil the content!**

Concludes the third iteration of the skinning contest by adding the newspost and updating the wiki

written by @zeusminus with graphics from Redo_

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: news/2022-10-20-skinning-contest-chromatic-alteration-results.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Attracting a total of **968** sour grape enjoyers, [sour_grapes](https://osu.ppy.sh/community/forums/topics/1626950) by [koomo](https://osu.ppy.sh/users/2168518) achieves the 3rd place!
+
+Adopting a protanopic palette, sour_grapes portrays a futuristic look with Ene from *Kagerou Project* on the side. Featuring matrix-esque panels and high-tech design from head to toe, the skin makes you feel like you're playing the game through a visor, while also not letting up on the minimalistic outlook. 
+
+Needless to say, if you're looking for a cutting-edge skin offering a *slow journey* into your osu! and osu!taiko ventures, sour_grapes is your go-to one!
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/70dYbj9y0qM" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Executing a masterpiece going all out on technological advancements, koomo gets to achieve the 3rd place profile badge with their sour_grapes entry! Congratulations!
```
Comment (Walavouchey) [inline news/2022-10-20-skinning-contest-chromatic-alteration-results.md:25]:
```suggestion
Executing a masterpiece going all out on technological advancements, koomo gets to achieve the 3rd-place profile badge with their sour_grapes entry! Congratulations!
```

File: news/2022-10-20-skinning-contest-chromatic-alteration-results.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+![](/wiki/shared/news/2022-10-20-skinning-contest-chromatic-alteration-results/grapes.png)
+
+## 2nd Place - Ralsei Dark
+
+Themed around objectively the cutest character in *Deltarune*, [Ralsei Dark](https://osu.ppy.sh/community/forums/topics/1629393) was favoured by **1119** voters, allowing [Corne2Plum3](https://osu.ppy.sh/users/15646039) to claim the 2nd place!
+
+Featuring a tritanopic palette, Ralsei Dark brings about the artsy look of the well-renowned *toby fox* games perfectly integrated into your osu! client. Furthermore, despite going all out on textures and art plastered everywhere, the skin actually manages to balance it out masterfully through rather minimalist outlook on gameplay and results interface.
```
Comment (Walavouchey) [inline news/2022-10-20-skinning-contest-chromatic-alteration-results.md:33]:
```suggestion
Featuring a tritanopic palette, Ralsei Dark brings about the artsy look of the well-renowned *toby fox* games perfectly integrated into your osu! client. Furthermore, despite going all out on textures and art plastered everywhere, the skin actually manages to balance it out masterfully through a rather minimalist outlook on gameplay and the results screen.
```

File: news/2022-10-20-skinning-contest-chromatic-alteration-results.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Thanks to their beloved work of art, Ralsei Dark, Corne2Plum3 is awarded with a cute Ralsei artwork immortalized in the form of a profile badge!
+
+![](/wiki/shared/news/2022-10-20-skinning-contest-chromatic-alteration-results/ralsei.png)
+
+## Winner - 霜滅 SOUMETSU
+
+With a grand total of **1497** votes, [霜滅 SOUMETSU](https://osu.ppy.sh/community/forums/topics/1628514) enables [thetasigma](https://osu.ppy.sh/users/6234482), a veteran runner-up from the previous iteration, to finally reach for the crown!
+
+Utilizing a protanopic palette, 霜滅 SOUMETSU combines the charm of a typical minimalist skin and a popular *Genshin Impact* character, Kamisato Ayaka. From adorable artworks to meticulously designed assets, the skin sets up a perfect example for its kind. To look at it from a different perspective, the artworks portraying traditional Japanese way of life embedded within the modern, minimalist textures essentially signifies the harmony between the two distinct elements.
```
Comment (Walavouchey) [inline news/2022-10-20-skinning-contest-chromatic-alteration-results.md:47]:
```suggestion
Utilising a protanopic palette, 霜滅 SOUMETSU combines the charm of a typical minimalist skin and a popular *Genshin Impact* character, Kamisato Ayaka. From adorable artworks to meticulously designed assets, the skin sets up a perfect example for its kind. To look at it from a different perspective, the artworks portraying traditional Japanese way of life embedded within the modern, minimalist textures essentially signifies the harmony between the two distinct elements.
```

File: news/2022-10-20-skinning-contest-chromatic-alteration-results.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Themed around objectively the cutest character in *Deltarune*, [Ralsei Dark](https://osu.ppy.sh/community/forums/topics/1629393) was favoured by **1119** voters, allowing [Corne2Plum3](https://osu.ppy.sh/users/15646039) to claim the 2nd place!
+
+Featuring a tritanopic palette, Ralsei Dark brings about the artsy look of the well-renowned *toby fox* games perfectly integrated into your osu! client. Furthermore, despite going all out on textures and art plastered everywhere, the skin actually manages to balance it out masterfully through rather minimalist outlook on gameplay and results interface.
+
+Well, it's a Ralsei-themed skin after all. Who wouldn't love it? Especially when all the artwork is created by the very hands of the skinner itself!
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/Y-x_M8Jnpt4" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Thanks to their beloved work of art, Ralsei Dark, Corne2Plum3 is awarded with a cute Ralsei artwork immortalized in the form of a profile badge!
```
Comment (Walavouchey) [inline news/2022-10-20-skinning-contest-chromatic-alteration-results.md:39]:
```suggestion
Thanks to their beloved work of art, Ralsei Dark, Corne2Plum3 is awarded with a cute Ralsei artwork immortalised in the form of a profile badge!
```

**Review by Walavouchey — COMMENTED** [review]
otherwise ok 👍

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**